winrt-related-src/midl-3/predefined-attributes.md

```diff
@@ -62,9 +62,23 @@ namespace BgLabelControlApp
 ```
 
 ## The `[contract]` attribute
-Do not use the `contract` attribute in your own APIs; it only has meaning for built-in Windows APIs.
 
-The `contract` attribute specifies the name and version of the Windows 10 API contract (see [Programming with extension SDKs](../extension-sdks/device-families-overview.md)) in which the attributed type and/or member was first introduced into Windows (hence it is not meaningful for APIs not delivered as part of Windows). The attribute takes the form `[contract(ContractName, ContractVersion)]`, and it appears before the item to which it applies.
+The `contract` attribute is used to version APIs; for example when creating a
+Software Development Kit where strong contracts are needed. It indicates the API contract
+name and two decimal values; `[contract(ContractName, LeftNumber.RightNumber)]`
+with `RightNumber` being optional. 
+
+It appears before the item to which it applies and can apply to several artifacts
+by using braces (`{}`) to group those parts of the API.
+
+Designs that don't need versioning should not use this attribute, or should keep the
```

Comment:
I think this is already covered in the opening paragraphs of this page.

> If you're a component developer and/or a library author, then you may wish to use these attributes to ensure that your components remain binary-stable from one version to the next.
> 
> If you're an application developer, then in general you won't need to use these attributes because you'll recompile your application after revisioning your types.